tests/rotate/test-rotate-end-to-end.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
```

Comment:
@LeeBergstrand I have used a shell script for end-to-end tests. Is this reasonable, or do you think something in Python (e.g., even integrated into the main Python package) would be more suitable? Thanks.

Reply:
@jmtsuji Given the number of test states and test files, I think this should be handled by unit testing of the underlying functions.

Reply:
@jmtsuji For example, rotate_sequence_to_position or rotate_sequence_to_fraction should be unit tested rather than tested end-to-end.

Reply:
Your end to end tests should basically be testing one or two states. Unit tests are used to test the underlying functions.